source/plugins.rst

```diff
@@ -4,6 +4,260 @@
+
+All plugins are located in the "Plug-Ins" submenu of the canvas.
+
+Rationale
+====
+
+People often write some scripts to make useful things on Synfig (sif) files. The most of these scripts are written in python. But for ordinary users running custom scripts from terminal is tricky. With plugins feature user can install scripts as easy as they copy files and transparently run them in the same way as they use standard Synfig Studio commands. Also, runing scripts from menu is much faster than from terminal and it greatly improves the workflow for advanced users. Having this feature allows to easily add simple functions to Synfig Studio by writting scripts in python.
+
+How to install plugins
+====
+
+Plugin is a directory, containing the python script (*.py file), plugin.xml and maybe some other files if they are required by python script.
+
+To install the plugin user should copy its directory into the following location:
+
+* **Windows Vista/7/8:** C:\Users\USERNAME\Synfig\plugins
+* **Windows XP:** C:\Documents and Settings\USERNAME\Synfig\plugins
+* **Linux:** ~/.config/synfig/plugins
+* **OSX:** /Users/USERNAME/Library/Synfig/plugins
+
+The system-wide location for the plugins is USER_DIRECTORY/SYNFIG_CONFIGURATION_DIR/plugins
+
+Where to find plugins
+----
+
+* `Yoyobuae:FreeForm Deform <https://github.com/yoyobuae/synfig-plugins>`_
+* `Morevna:Plugins feature in Synfig Studio <http://morevnaproject.org/2012/06/11/plugins-feature-in-synfig-studio/>`_
+* `Berteh:Import Labels and Timings <https://github.com/berteh/synfig-import-labels>`_
+* `Synfig forum: scripts/plugin thread <https://forums.synfig.org/c/development/scripting>`_
+
+Plugin structure
+====
+
+A plugin keeps all its files inside a directory as described above.
+
+Synfig parses the file called plugin.xml found in each plugin directory, which defines plugin metadata and how to invoke the script.
+
+This section describes the xml elements available and what they do.
+
+<plugin>
+----
+
+Root element. Can contain the following:
+
+* <name> The name of the plugin, must have at least the default version
+* <exec> Script to run when the user clicks on the Plugin menu. Can be omitted if there are importers or exporters
+* <exporter> Defines an exporter, a plugin can have multiple of these
+* <importer> Defines an importer, a plugin can have multiple of these
+
+<name>
+----
+
+Name of the plugin, can be specified multiple times to provide translations
+
+
+Example:
+::
+    <name>This is the default name</name>
+    <name xml:lang="it">This is name it will show if you set Italian as your language</name>
+
+<exec>
+----
+
+Defines a script to run, the text contents must be a path (relative to the plugin directory) of a script to run.
+
+If present inside <plugin>, the plugin will be shown in the Plugins menu, and the script will be invoked when you click on the corresponding menu item.
+
+It has a few attributes, all optional.
+
+{|class=wikitable
```

Comment:
I suggest to replace wikitable with Lists (Enumerated List or Definition List) - https://sphinx-rtd-theme.readthedocs.io/en/stable/demo/lists_tables.html#enumerated-lists